atest/acceptance/keywords/content_assertions.robot

```diff
@@ -47,14 +73,30 @@ Page Should Contain With Custom Log Level DEBUG
 
 Page Should Contain With Custom Log Level TRACE
```

Comment:
This test is now basically duplicate for "Page Should Contain Using Default Custom Log Level' on line 30.
For consistency would it not make more sense to use the title from this test("Page Should Contain With Custom Log Level Trace")? but use the re-setting of original Log Level like in the other tests? `${old_level}= Set Log Level TRACE`

Reply:
This is a very good catch and great feedback. There is a subtle difference in the tests. The first doesn't set the loglevel (Line 39*: `Page Should Contain non existing text`). Whereas the second explicitly specifies the loglevel to be TRACE (Line 83*: `    Page Should Contain    non existing text    TRACE`). It took me a while to go back through this code and understand the changes I had initially made here. I recalled something about the default level and how the doc specified one level but execution said another. Taking your feedback I have updated the documentation on a few of these tests. Let me know if this is any clearer.

* lines numbers are reference updated code [27e0602](https://github.com/robotframework/SeleniumLibrary/pull/1814/commits/27e06024cd75c18f2a93bfe31bcc4f29da761c26)